_posts/blog/2017-07-19-Actions-for-android-devs.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Google Assistant Development for Android Devs (Part 1)"
+author: Patrick Jackson
+categories: Development
+tags: [Google Assistant, Actions on Google, conversational interfaces, Android, Kotlin]
+---
+
+
+# Google Assistant Development for Android Devs (Part 1)
+
+![](https://developers.google.com/actions/images/surfaces-hero.svg)
```

Comment:
Maybe we can host this somewhere else to avoid hot-linking to a Google image that may change?
Also, maybe we can make a custom version and use that as the "featured image" that gets used when shared on FB etc. I think you can probably find a designer to do that for you. :D